resources/semiology_dictionary.yaml

```diff
@@ -496,15 +496,17 @@
 # negations:
 #   "No": [" no ", "none", "denies?", "deny", "absent", "without"]
 
-Hippocampal Sclerosis:
-  [
-    "(?<!no )hippocampal sclerosis",
-    " HS ",
-    "(?<!no )hipp?o(camp)? scl",
-    "mesial scl",
-    "temporal scl",
-    "mTL scl",
-  ]
+# Hippocampal Sclerosis:
```

Comment:
Can you just remove the lines? This is noise in the file and in the repository. You can always come back to a previous version, everything is stored in the repo history.